src/scripting/effects.cpp

```diff
@@ -1056,25 +945,21 @@ uint32_t es_x_substate_scope(EFFECT_PARAMTERS) {
 		}
 		return 0;
 	} else {
-		std::vector<dcon::nation_id> nlist;
-
+		uint32_t i = 0;
```

Comment:
I don't understand this change. This case is supposed to handle picking a random substate, and it appears to no longer do that

Reply:
random handled above

Reply:
ok